### PR TITLE
dbmate 1.5.0 (new formula)

### PR DESCRIPTION
**Pull request by amacneil**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dbmate.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  head "https://github.com/amacneil/dbmate.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s", "-o", bin/"dbmate", "."
+  end
+
+  test do
+    system bin/"dbmate", "-v"
```
Comment (zbeekman) [inline Formula/dbmate.rb]: We need a better test here. This is not sufficient to ensure correct functionality.

Comment (zbeekman) [inline Formula/dbmate.rb]: We need a better test here, this is not sufficient to ensure proper functionality. A test should exercise some minimal functionality. You can check for the existence and contents of log files, perform some simple action testing post- (and maybe pre-) conditions, and/or parse stdout/stderr for actual content indicating it works as expected.

**Review by zbeekman — CHANGES_REQUESTED** [review]
Needs better test

File: Formula/dbmate.rb (hunk 1 above)
Comment (amacneil) [reply Formula/dbmate.rb]: Thanks for the feedback. I updated with a test of actual functionality.

**Review by zbeekman — APPROVED** [review]

**Comment by zbeekman** [comment]
Speaking of migration: Once this is merged, will you mark it as migrated to homebrew-core in your tap? Or do you plan to keep maintaining the tap? (Just curious and I want to make sure you know how to mark it as migrated for existing users, if you want to do that.)

**Comment by zbeekman** [comment]
(@amacneil To do that, you just need to add a `tap_migrations.json` file to your tap. See https://github.com/Homebrew/homebrew-core/blob/master/tap_migrations.json)

**Comment by amacneil** [comment]
Thanks - yes, I plan to migrate. I wasn't aware of that feature, but it looks perfect.

Is this sufficient? https://github.com/amacneil/homebrew-dbmate/pull/1

**Comment by zbeekman** [comment]
Thanks for your contribution to Homebrew! 🎉 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**